includes/wikia/tasks/Tasks/MultiTask.class.php

```diff
@@ -199,19 +259,19 @@ private function runOnWikis($wikiInputRaw, $lang, $cat, $wikiId, $action, $comma
 			$query->AND_( 'city_list.city_lang' )->EQUAL_TO( $lang );
 		}
 
-		if (!empty($wikiId)) {
-			$query->AND_('city_list.city_id')->EQUAL_TO($wikiId);
+		if ( !empty( $wikiId ) ) {
```

Comment:
1. s/wikiId/wikisId/

2. This entire PR purpose is to drop support for processing all wikis. Maybe we should fail early here with a descriptive error message if we discover $wikisId is empty?